### PR TITLE
Implement statistics page

### PR DESCRIPTION
**Pull request by kantoniak**
* Implement adding/removing datasets
* Handle network operations
* Add grouping by year/month etc.
* Add moving between datespans
* Add Polish and English translations (currently hardcoded to use English)

Preview: https://eaterslab-dev.herokuapp.com/admin/stats/
statistics-2

### CODE REVIEW

File: django-server/server/admin/static/admin/stats.js [hunk 1]
```diff
@@ -0,0 +1,873 @@
...
+}
+
+function hourLeadZero(date) {
+  return ('0' + date.getHours()).slice(-2);
+}
+
+function minuteLeadZero(date) {
+  return ('0' + date.getMinutes()).slice(-2);
+}
+
```
Comment (avan1235) [inline django-server/server/admin/static/admin/stats.js:97]: create single function that is called by these 3

File: django-server/server/admin/static/admin/stats.js [hunk 2]
```diff
@@ -0,0 +1,873 @@
...
+    this.g = g;
+    this.b = b;
+    this.h = h;
+    this.s = s;
+    this.l = l;
+  }
+
+  rgba(alpha) {
+    return 'rgba(' + this.r + ', ' + this.g + ', ' + this.b + ', ' + alpha + ')';
+  }
```
Comment (avan1235) [inline django-server/server/admin/static/admin/stats.js:125]: what standard of JS do you use. If some newer let's use [templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

File: django-server/server/admin/static/admin/stats.js [hunk 3]
```diff
@@ -0,0 +1,873 @@
...
+function dateYM(date) {
+  return date.getFullYear() + '-' + monthLeadZero(date);
+}
+
+function dateYMD(date) {
+  return date.getFullYear() + '-' + monthLeadZero(date) + '-' + dayLeadZero(date);
+}
+
+function dateYMDHM(date) {
+  return date.getFullYear() + '-' + monthLeadZero(date) + '-' + dayLeadZero(date) + ' ' + hourLeadZero(date) + ':' + minuteLeadZero(date);
```
Comment (avan1235) [inline django-server/server/admin/static/admin/stats.js:107]: see comment below about templates

File: django-server/server/admin/static/admin/stats.js [hunk 4]
```diff
@@ -0,0 +1,873 @@
...
+  }
+
+  rgba(alpha) {
+    return 'rgba(' + this.r + ', ' + this.g + ', ' + this.b + ', ' + alpha + ')';
+  }
+
+  hsla(alpha) {
+    return 'hsla(' + this.h + ', ' + this.s + '%, ' + this.l + '%, ' + alpha + ')';
+  }
+}
```
Comment (avan1235) [inline django-server/server/admin/static/admin/stats.js:130]: just FYI to remember about templates

File: django-server/server/admin/static/admin/stats.js [hunk 5]
```diff
@@ -0,0 +1,873 @@
...
+  /*
+  fetch(API_URL_BASE + 'cafeterias/' + d + '/occupancy/?from=' + from.toISOString() + '&to=' + to.toISOString())
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error('HTTP status ' + response.status);
+      }
+    })
+  */
```
Comment (avan1235) [inline django-server/server/admin/static/admin/stats.js:708]: make an issue for this maybe not to forget

Comment (kantoniak) [reply django-server/server/admin/static/admin/stats.js:705]: Referencing #141 here so it shows up in there.

**Pull request merged**